src/price.css

```diff
@@ -0,0 +1,18 @@
+.price {
+  display: flex;
+  align-items: center;
+  margin-top: 23px;
+  margin-bottom: 16px;
+}
+
+.price__text {
```

Comment:
Consider renaming the class 'price__text' to 'product-card__price-text' to maintain consistency with BEM naming conventions.